packages/app/src/plugins.ts

```diff
@@ -18,7 +18,7 @@ export { plugin as LighthousePlugin } from '@backstage/plugin-lighthouse';
 export { plugin as CatalogPlugin } from '@backstage/plugin-catalog';
 export { plugin as ScaffolderPlugin } from '@backstage/plugin-scaffolder';
 export { plugin as TechRadar } from '@backstage/plugin-tech-radar';
-export { plugin as Explore } from '@backstage/plugin-explore';
+export { explorePlugin } from '@backstage/plugin-explore';
```

Comment:
Nit - Looks a bit different than the rest. :)

Reply:
Followed this: 

![image](https://user-images.githubusercontent.com/648527/106250711-4f0c3480-6214-11eb-8085-76ffe91f656b.png)

https://backstage.io/docs/plugins/composability#naming-patterns

Reply:
Aha! Thanks, I didn't realize the naming pattern change. 🙏

Reply:
![image](https://user-images.githubusercontent.com/3097461/106253134-639dfc00-6217-11eb-9d85-e7a184fca14a.png)